recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - python >=3.8
     - click >=7.0
     - cloudpickle >=1.5.0
-    - cytoolz >=0.10.0
+    - cytoolz >=0.10.1
```

Comment:
Seems like this was a change made with https://github.com/dask/distributed/pull/7285 that we missed in the 2023.1.0 release? cc @jrbourbeau @jakirkham just making sure that leaving this at `>=0.10.0` wasn't intentional in the past few releases.

Shouldn't be difficult to submit a repodata patch to update this for the impacted versions

Reply:
Should we bump the [`toolz` version in `requirements.txt`]( https://github.com/dask/distributed/blob/f4328bbd919a08fc351a7514bf7895e14a5de7a4/requirements.txt#L12 ) as well?

Reply:
Not immediately obvious to me? It doesn't look like there's a package for `toolz` 0.10.1 available on either PyPI or conda-forge, and it looks like Distributed's mindeps testing is currently running without issues pinned to 0.10.0 (though not sure if this is case where installing `cytoolz` in the same environment would override this)

Reply:
Gotcha. Yeah guess `0.11.0` is the next version where they are aligned. So sounds like we leave `toolz` alone